app/serializers/disability_claim_detail_serializer.rb

```diff
@@ -34,6 +34,28 @@ def phase
     phase_from_keys 'claimPhaseDates', 'latestPhaseType'
   end
 
+  def tracked_items
+    items = []
+    items_src = object.data['consolidatedTrackedItemsList'] || []
```

Comment:
Yes. I think what I will do is make it so that all items are passed through, including items without dates. Those can just go to the end of the timeline.